Source/gg2/Scripts/Rewards/draw_name_badges.gml

```diff
@@ -13,8 +14,11 @@ draw_set_halign(fa_left);
 var i;
 for (i = 0; i < ds_list_size(player.badges); i += 1)
 {
-    draw_sprite(HaxxyBadgeS, ds_list_find_value(player.badges, i), _x, _y - 1);
+    //seems a bit overkill but its the easiest way to do draw_sprite transparency from what i can gather
+    draw_sprite_ext(HaxxyBadgeS, ds_list_find_value(player.badges, i) ,_x, _y - 1, 1, 1, 0, c_white, alpha);
     _x += sprite_get_width(HaxxyBadgeS);
 }
 
+draw_set_alpha(alpha);
 draw_text(_x, _y, player.name);
+draw_set_alpha(0);

```

Comment:
If you're going to set alpha to anything at the end, it should be whatever alpha you had before, right? You can get the current alpha with `draw_get_alpha()`.